components/SummaryComponents/SummaryContainer.tsx

```diff
@@ -99,7 +105,7 @@ export default function SummaryContainer(props: {
         <Grid>
           <h1 className={styles.noMargins}><ForestIcon /> Evergreen Dashboard</h1>
           <p className={styles.subtitle}>
-            Monitoring dependencies for <b className={styles.orgTitle}>{props.targetOrganisation}</b> Github Organisation
+            Monitoring dependencies for the <b className={styles.orgTitle}><a href={props.auxData.orgLink}>{props.auxData.orgName}</a></b> Github Organisation
```

Comment:
Not sure exactly how to fix this but when you have visited the link before it makes it purple which is really hard to see with the black background.
![Screen Shot 2022-10-11 at 8 29 40 am](https://user-images.githubusercontent.com/81656054/194955334-0bc1a993-e46a-4036-b0c6-c56dc5964bac.png)